packages/block-directory/src/store/selectors.js

```diff
@@ -50,11 +50,15 @@ export function getInstalledBlockTypes( state ) {
  * Returns true if application is calling install endpoint.
  *
  * @param {Object} state Global application state.
+ * @param {string} blockId Id of the block.
  *
  * @return {boolean} Whether its currently installing
  */
-export function isInstalling( state ) {
-	return state.blockManagement.isInstalling;
+export function isInstalling( state, blockId ) {
+	if ( ! state.blockManagement.isInstalling[ blockId ] ) {
+		return false;
+	}
+	return state.blockManagement.isInstalling[ blockId ];
```

Comment:
Funny enough, I had it that way but switched it back since it diverges from the way its done in other selectors.

Additionally, I find it less 'readable' for people who don't understand JS quirks.

Reply:
huh, I found _this_ pattern in a few other selectors. I don't have any strong preference, so this is good 👍